docs/HowTo/Monitor/Logging.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Teku log level settings and log formatting
+---
+
+# Logging
+
+Teku uses Log4J2 for logging, and provides multiple methods to configure logging behavior:
+
+* [Basic](#basic-log-level-setting) - changes the log level.
+* [Destination logging](#configure-log-destination) - configure the destination for log output.
+* [Custom logging](#advanced-custom-logging) - custom logging to configure the output and format
+    of the logs.
+
+## Basic log level setting
+
+Use the [`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) command line option to specify
+logging verbosity. The [`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) option changes the
+volume of events displayed in the log. Valid log levels are `OFF`, `FATAL`, `ERROR`, `WARN`,
+`INFO`, `DEBUG`, `TRACE`, `ALL`. The default level is `INFO`.
+
+For most use-cases, the basic method provides enough configurability.
```

Comment:
I'm not sure this is true for Teku - changing the log destination is likely to be very common.

Reply:
We also have `--log-color-enabled`, `--log-include-events-enabled` (to print slot & epoch events) and `--log-include-validator-duties-enabled` to enable logs when validators perform duties which provide finer grained control over the log level.

Reply:
Done